pkg/apps/blackduck/latest/creater.go

```diff
@@ -179,8 +178,9 @@ func (hc *Creater) Ensure(blackduck *blackduckapi.Blackduck) error {
 			}
 		}
 
-		if err := util.ValidatePodsAreRunningInNamespace(hc.kubeClient, blackduck.Spec.Namespace, 600); err != nil {
-			return err
+		err = util.WaitUntilPodsAreReady(hc.kubeClient, blackduck.Spec.Namespace, fmt.Sprintf("app=%s,name=%s,component notin (postgres,cfssl,configmap,serviceAccount)", util.BlackDuckName, blackduck.Name), hc.config.PodWaitTimeoutSeconds)
```

Comment:
configmap, service account can be removed from the in clause

Reply:
Done